ftp.go

```diff
@@ -188,13 +203,15 @@ func DialWithContext(ctx context.Context) DialOption {
 }
 
 // DialWithTLS returns a DialOption that configures the ServerConn with specified TLS config
+// the explicit parameter means the client first runs an explicit command ("AUTH TLS")
 //
 // If called together with the DialWithDialFunc option, the DialWithDialFunc function
 // will be used when dialing new connections but regardless of the function,
 // the connection will be treated as a TLS connection.
-func DialWithTLS(tlsConfig *tls.Config) DialOption {
+func DialWithTLS(tlsConfig *tls.Config, explicit ...bool) DialOption {
```

Comment:
why a variadic of bool and not just one bool ?

Reply:
In order not to bring breaking changes to the API.